Source/Common/FEXServerClient.cpp

```diff
@@ -121,7 +121,7 @@ fextl::string GetServerMountFolder() {
     // This is due to pressure-vesssel being a chroot environment.
     // It by default maps the host-filesystem to `/run/host/` so we need to redirect.
     // After pressure-vessel is fully set up it will set the `FEX_ROOTFS` environment variable,
-    // which the FEXInterpreter will pick up on.
+    // which the FEX will pick up on.
```

Comment:
Using an article here is awkward when just referring to "FEX".
```suggestion
    // which FEX will pick up.
```